apps/nextjs-website/src/components/atoms/GuideMenu/GuideMenu.tsx

```diff
@@ -9,12 +21,109 @@ type GuideMenuProps = {
 };
 
 const components: RenderingComponents<ReactNode> = {
-  Link: ({ href, children }) => <a href={href}>{children}</a>,
-  Item: ({ isLeaf, children }) => (
-    <li className={isLeaf ? 'leaf' : 'node'}>{children}</li>
+  Link: ({ href, children }: any) => {
+    const router = useRouter();
+    const isSelected = useMemo(() => {
+      const lastPathSegment = href?.split("/")[href?.split("/").length - 1];
+      const routerLastPathSegment =
+        router.asPath?.split("/")[router.asPath?.split("/").length - 1];
+
+      return lastPathSegment === routerLastPathSegment;
+    }, [children, href, router.asPath]);
+    return (
+      <ListItemButton
+        component="a"
+        href={href}
+        style={{
+          width: "100%",
+          backgroundColor: isSelected ? "rgba(0, 115, 230, 0.08)" : undefined,
+          position: "relative",
+        }}
+      >
+        {isSelected && (
+          <div
+            style={{
+              position: "absolute",
+              left: -300,
+              width: 300,
+              height: "100%",
+              backgroundColor: "rgba(0, 115, 230, 0.08)",
+            }}
+          />
+        )}
+
+        <Typography color={isSelected ? "#0062C3" : "black"}>
+          {children}
+        </Typography>
+
+        {isSelected && (
+          <div
+            style={{
+              position: "absolute",
+              right: 0,
+              width: 2,
+              height: "100%",
+              backgroundColor: "rgba(0, 98, 195, 1)",
+            }}
+          />
+        )}
+      </ListItemButton>
+    );
+  },
+  Item: ({ isLeaf, children }: any) => {
+    const router = useRouter();
+
+    return (
+      <ListItem
+        className={isLeaf ? "leaf" : "node"}
+        style={{
+          display: "flex",
+          flexDirection: "column",
+          alignItems: "flex-start",
+          padding: 0,
+          paddingLeft: "1rem",
+        }}
+      >
+        {isLeaf ? (
+          children
+        ) : (
+          <Accordion
+            style={{
+              background: "transparent",
+              width: "100%",
+              paddingRight: 0,
+            }}
+            defaultExpanded={router.asPath.includes(children[0].props.href)}
+          >
+            <AccordionSummary
+              expandIcon={<ExpandMoreIcon />}
+              style={{ height: 60 }}
+            >
+              {children[0]}
+            </AccordionSummary>
+            <AccordionDetails style={{ padding: 0 }}>
+              {children.slice(1)}
+            </AccordionDetails>
+          </Accordion>
+        )}
+      </ListItem>
+    );
+  },
+  List: ({ children }: any) => <List>{children}</List>,
+  Title: ({ children }: any) => (
+    <Typography
+      component={"h2"}
+      style={{
+        color: "#5C6F82",
```

Comment:
Assuming the value `#5C6F82` exists inside the `theme`, we can take it from there

Reply:
It does not, its also used as hex value in GuidesSection.txt and [...productGuidePage.tsx]